processor/transformprocessor/README.md

```diff
@@ -169,10 +173,18 @@ Since the transform processor utilizes the OTTL's contexts for Traces, Metrics,
 In addition to OTTL functions, the processor defines its own functions to help with transformations specific to this processor:
 
 **Metrics only functions**
-- [convert_sum_to_gauge](#convert_sum_to_gauge)
-- [convert_gauge_to_sum](#convert_gauge_to_sum)
-- [convert_summary_count_val_to_sum](#convert_summary_count_val_to_sum)
-- [convert_summary_sum_val_to_sum](#convert_summary_sum_val_to_sum)
+- [Transform Processor](#transform-processor)
```

Comment:
I don't think this section should be modified.

Reply:
It definitely should not.... not sure what happened here

Reply:
Fixed